frameworks/core_foundation/views/view/enabled.js

```diff
@@ -204,6 +204,7 @@ SC.View.reopen(
     // Update the display if in a visible state.
     switch (state) {
     case SC.CoreView.ATTACHED_SHOWN:
+    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
```

Comment:
I know, but I would suggest to add this comment on every spot where it happens, just to have it stick out, indicating that it has been done on purpose, and not coincidentally.